test/index.test.ts

```diff
@@ -58,6 +58,15 @@ test("README example for app auth", async () => {
   });
 });
 
+test("throws if invalid 'type' is provided", async () => {
+  const auth = createAppAuth({
+    appId: APP_ID,
+    privateKey: PRIVATE_KEY,
+  });
+
+  expect(auth({ type: "app2" })).toThrow();
```

Comment:
This does not compile:

![image](https://user-images.githubusercontent.com/2574275/106674115-fa214300-65b2-11eb-80d0-51dd717bba9e.png)

---

![image](https://user-images.githubusercontent.com/2574275/106674131-01485100-65b3-11eb-9df7-c8557dc3a350.png)

Reply:
add a [`@ts-expect-error` comment](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments)

Reply:
```suggestion
  // @ts-expect-error TS2322
  expect(auth({ type: "app2" })).toThrow();
```

compare https://github.com/octokit/webhooks.js/search?q=ts-expect-error